tomopy/prep/normalize.py

```diff
@@ -119,32 +120,23 @@ def normalize(arr, flat, dark, cutoff=None, ncore=None, out=None):
         Normalized 3D tomographic data.
     """
     arr = dtype.as_float32(arr)
-    flat = dtype.as_float32(flat)
-    dark = dtype.as_float32(dark)
-
-    flat = flat.mean(axis=0)
-    dark = dark.mean(axis=0)
-
-    arr = mproc.distribute_jobs(
-        arr,
-        func=_normalize,
-        args=(flat, dark, cutoff),
-        axis=0,
-        ncore=ncore,
-        nchunk=0,
-        out=out)
-    return arr
-
-# in-place normalization
-def _normalize(proj, flat, dark, cutoff):
-    denom = flat - dark
-    denom[denom < 1e-6] = 1e-6
-    proj -= dark
-    np.true_divide(proj, denom, proj)
+    flat = np.mean(flat, axis=0, dtype=np.float32)
+    dark = np.mean(dark, axis=0, dtype=np.float32)
+    
+    if ncore:
```

Comment:
I wasn't sure what would happen if you call `set_num_threads` with `None` as argument. I just checked: it throws a `TypeError`. So we could keep the `if` or change it to a `try...catch` construction. Do you prefer one over the other?